backend/src/resolvers/socialMedia.spec.js

```diff
@@ -30,20 +39,63 @@ describe('CreateSocialMedia', () => {
     await factory.cleanDatabase()
   })
 
+  describe('unauthenticated', () => {
+    it('throws authorization error', async () => {
+      client = new GraphQLClient(host)
+      const variables = { url: 'http://nsosp.org' }
+      await expect(
+        client.request(mutationC, variables)
+      ).rejects.toThrow('Not Authorised')
+    })
+  })
+
   describe('authenticated', () => {
     beforeEach(async () => {
       headers = await login({ email: 'test@example.org', password: '1234' })
       client = new GraphQLClient(host, { headers })
     })
 
+    it('creates social media with correct URL', async () => {
+      const variables = { url: 'http://nsosp.org' }
+      await expect(
+        client.request(mutationC, variables)
+      ).resolves.toEqual(expect.objectContaining({
+        CreateSocialMedia: {
+          id: expect.any(String),
+          url: 'http://nsosp.org'
+        }
+      }))
+    })
+
+    it('deletes social media', async () => {
+      const variablesC = { url: 'http://nsosp.org' }
```

Comment:
maybe, we could find a more descriptive name? 
what about `deletionVariables`

Reply:
Thank you @mattwr18 for the hint. I have done.